Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -101,14 +113,18 @@ struct FPTIBatchData: Codable {
 
         enum CodingKeys: String, CodingKey {
             case appSwitchURL = "url"
+            case buttonOrder = "button_order"
```

Comment:
Based on slack convo:
```suggestion
            case buttonOrder = "button_position"
```